.github/DEVELOPMENT.md

```diff
@@ -4,184 +4,40 @@ This page contains steps to build and run the .NET MAUI repository from source.
-   - [macOS](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)  
-2. If you're on a Windows development machine, install [SDK 20348](https://go.microsoft.com/fwlink/?linkid=2164145)
-3. If you're on a MacOS development machine, install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-on-macos)
+### Visual Studio
+Follow the instructions here to install .NET MAUI with Visual Studio Stable:
+   - [Windows](https://learn.microsoft.com/dotnet/maui/get-started/installation?tabs=vswin)
+      - Select the 20348 SDK option inside Individual Components or [install 20348 manually](https://go.microsoft.com/fwlink/?linkid=2164145)
+      - If you know you have 20348 installed but are still getting an error around this SDK missing, trying uninstalling and reinstalling the SDK.
+   - [macOS](https://learn.microsoft.com/dotnet/maui/get-started/installation?tabs=vsmac)  
    
 ### iOS / MacCatalyst
 
-iOS and MacCatalyst will require Xcode 13.3 Stable. You can get this [here](https://developer.apple.com/download/more/?name=Xcode).
+iOS and MacCatalyst will require current stable Xcode. You can get this [here](https://developer.apple.com/download/more/?name=Xcode).
 
 ### Android
 
-Android API-31 (Android 12) is now the default in .NET 6.
-
-## Running
-
-### Compile using a local `bin\dotnet`
-
-This method ensures that the workloads installed by Visual Studio won't get changed. This is usually the best method to use if you want to preserve the global state of your machine. This method will also use the versions that are specific to the branch you are on which is a good way to ensure compatibility.
-
-#### Cake
-
-You can run a `Cake` target to bootstrap .NET 6 in `bin\dotnet` and launch Visual Studio:
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --target=VS
-```
-
-#### Testing branch against your project
-`--sln=<Path to SLN>`
-- This will pack .NET and then open a VS instance using the local pack. This is useful if you want to check to see if the changes in a branch will address your particular issues. Pack only runs the first time so you will need to explicitly add the `--pack` flag if you make changes and need to repack.
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --sln="<download_directory>\MauiApp2\MauiApp2.sln" --target=VS
-```
-
-#### Pack
-`--pack`
-- This creates .NET MAUI packs inside the local dotnet install. This lets you use the CLI commands with the local dotnet to create/deploy with any changes that have been made on that branch (including template changes).
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --target=VS --pack --sln="<download_directory>\MauiApp2\MauiApp2.sln"
-```
+If you're missing any of the Android SDKs, Visual Studio should prompt you to intall them. If it doesn't prompt you then use the [Android SDK Manager](https://learn.microsoft.com/en-us/xamarin/android/get-started/installation/android-sdk) to install the necessary SDKs.
```

Comment:
```suggestion
If you're missing any of the Android SDKs, Visual Studio should prompt you to install them. If it doesn't prompt you then use the [Android SDK Manager](https://learn.microsoft.com/xamarin/android/get-started/installation/android-sdk) to install the necessary SDKs.
```

Reply:
Dangit! I thought I got them all this time